hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -3807,10 +3688,14 @@ public SecureBulkLoadManager getSecureBulkLoadManager() {
   }
 
   @Override
-  public EntityLock regionLock(List<RegionInfo> regionInfos, String description, Abortable abort)
-      throws IOException {
-    return new LockServiceClient(conf, lockStub, asyncClusterConnection.getNonceGenerator())
-      .regionLock(regionInfos, description, abort);
+  public EntityLock regionLock(
+      final List<RegionInfo> regionInfo,
+      final String description,
+      final Abortable abort
+  ) {
```

Comment:
Looks like this is use a different code formatter, should we use unified?

Reply:
I'm using the code formatter in `dev-support/hbase_eclipse_formatter.xml` and checkstyle doesn't complain about it. The newlines between method arguments make for simpler reading and shorter diffs in the long run. My rule of thumb is that if the parameter list need to wrap, they should probably be broken out into one line each.

What formatter do you use? It would be great if we didn't have to talk about this at all, if there was a tool like rust or go that just does it.

Reply:
OK, i am also use the code formatter in dev-support/hbase_eclipse_formatter.xml with intellij idea, but it does not format the code into it.

Reply:
Yeah, code looks unusual. See code around you.